src/core/annotation.js

```diff
@@ -866,15 +876,7 @@ var FileAttachmentAnnotation = (function FileAttachmentAnnotationClosure() {
 
     this.data.annotationType = AnnotationType.FILEATTACHMENT;
     this.data.file = file.serializable;
-
-    if (!dict.has('C')) {
-      // Fall back to the default background color.
-      this.data.color = null;
-    }
-
-    this.data.hasPopup = dict.has('Popup');
-    this.data.title = stringToPDFString(dict.get('T') || '');
-    this.data.contents = stringToPDFString(dict.get('Contents') || '');
+    this._preparePopup(dict);
```

Comment:
Nit: I think that you can now remove the variable and simply use parameters.dict here.